nomad/alloc_endpoint_test.go

```diff
@@ -77,6 +79,62 @@ func TestAllocEndpoint_List(t *testing.T) {
 	}
 }
 
+func TestAllocEndpoint_List_ACL(t *testing.T) {
+	t.Parallel()
+	s1, root := testACLServer(t, nil)
+	defer s1.Shutdown()
+	codec := rpcClient(t, s1)
+	testutil.WaitForLeader(t, s1.RPC)
+	assert := assert.New(t)
+
+	// Create the deployment
+	alloc := mock.Alloc()
+	allocs := []*structs.Allocation{alloc}
+	summary := mock.JobSummary(alloc.JobID)
+	state := s1.fsm.State()
+
+	assert.Nil(state.UpsertJobSummary(999, summary), "UpsertJobSummary")
+	assert.Nil(state.UpsertAllocs(1000, allocs), "UpsertAllocs")
+
+	stubAllocs := []*structs.AllocListStub{alloc.Stub()}
+	stubAllocs[0].CreateIndex = 1000
+	stubAllocs[0].ModifyIndex = 1000
+
+	// Create the namespace policy and tokens
+	goodToken := CreatePolicyAndToken(t, state, 1001, "test-good",
```

Comment:
@chelseakomlo had the comment to move towards `validToken` and `invalidToken`, with the policy names `test-valid`, `test-invalid`